app/controllers/annotations_controller.rb

```diff
@@ -119,6 +119,11 @@ def doc_annotations_list_view
 	# POST /annotations
 	# POST /annotations.json
 	def create
+		# token authentication
+		if api_call_from_non_textae?
+			return unless token_authenticated?
```

Comment:
`return`だけだとレンダリングしているか不安になります。
```suggestion
			return  render_token_invalid_error unless token_authenticated?
```
と、すると、レンダリングしていることが明確になって安心感が増すと思います。

Reply:
確かにそのとおりですね..!
修正しました。
https://github.com/pubannotation/pubannotation/pull/136/commits/7b2c82e9c9c628ff98fcbc48c01927465857e943